File: TCS CodeVita/encryptString.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    string s = "Attack cnsid sdcsivn sdjkjkcsdj";
    string sub = "";
    vector<string> temp;
    for (int i = 0; i < s.length(); i++)
    {
        if (sub.length() == 6)
        {
            temp.push_back(sub);
            sub = "";
        }
        if (s[i] == ' ')
        {
            sub += '*';
        }
        else
        {
            sub += s[i];
        }
    }

    if (sub.length() != 0)
    {
        for (int i = sub.length(); i < 6; i++)
        {
            sub += '.';
        }
        temp.push_back(sub);
    }

    string res = "";

    for (int j = 0; j < 6; j++)
    {
        for (int i = 0; i < temp.size(); i++)
        {
            res += temp[i][j];
        }
    }

    cout << res << endl;

    return 0;
}

File: TCS CodeVita/encryptString.py
s = "Attack cnsid sdcsivn sdjkjkcsdj"

sub = ''

l = []
for c in s:
    if(len(sub) == 6):
        l.append(sub)
        sub = ''
    if(c == ' '):
        sub += '*'
    else:
        sub += c


if(len(sub) != 0):
    for i in range(len(sub), 6):
        sub += '.'
    l.append(sub)


print(l)
